lib/services/access_list_test.go

```diff
@@ -293,12 +296,32 @@ func (t *testMembersGetter) GetAccessListMember(ctx context.Context, accessList
 	return member, nil
 }
 
+// GetLock gets a lock by name.
+func (t *testMembersAndLockGetter) GetLock(_ context.Context, name string) (types.Lock, error) {
+	if t.locks == nil {
+		return nil, trace.NotFound("not found")
+	}
+
+	lock, ok := t.locks[name]
+	if !ok {
+		return nil, trace.NotFound("not found")
+	}
+
+	return lock, nil
+}
+
+// GetLocks gets all/in-force locks that match at least one of the targets when specified.
+func (t *testMembersAndLockGetter) GetLocks(ctx context.Context, inForceOnly bool, targets ...types.LockTarget) ([]types.Lock, error) {
+	return maps.Values(t.locks), nil
```

Comment:
I might consider writing the naive loop here.

`maps.Values` was implemented in the proposal (the `x/exp` package) but when that package made its way into the standard library (`package maps`) it was removed. Sounds like it will be replaced with some sort of iterator in the future.